Cargo.toml

```diff
@@ -83,6 +83,7 @@ version_check = "0.9.4"
 xxhash-rust = { version = "0.8.6", features = ["xxh3"] }
 zstd = "0.13"
 uuid = { version = "1.7.0", features = ["v4"] }
+arboard = { version = "3.3.2" }
```

Comment:
Please use `optional-features = false`, we don't need the image copy/paste capabilities - it pulls in a lot of extra dependencies.